x-pack/plugins/security/public/management/users/users_management_app.test.tsx

```diff
@@ -22,16 +26,19 @@ describe('usersManagementApp', () => {
     const setBreadcrumbs = jest.fn();
     const history = scopedHistoryMock.create({ pathname: '/create' });
 
-    const unmount = await usersManagementApp.create({ authc, getStartServices }).mount({
-      basePath: '/',
-      element,
-      setBreadcrumbs,
-      history,
+    let unmount: Unmount = noop;
```

Comment:
__question:__ just for my understanding, why do we need to make this change (we only removed `coreStart.chrome.docTitle.reset();` in the `users_management_app.tsx`)?

Reply:
When I was fixing the unit test, there was a test failing saying that we were calling `coreStart.chrome.docTitle.reset()` twice during unmount instead of one time, I realized that during the unmount we were going in this condition https://github.com/elastic/kibana/blob/main/x-pack/plugins/security/public/components/breadcrumb.tsx#L150-L151. So I was able to remove redundant code.

Reply:
Ah, got it, thanks for clarifying!